### PR TITLE
Differentiate intent type comparisons for iOS 13

### PR DESCRIPTION
**Pull request by bhuangy**
Include fix in providerConfiguration for video option

### CODE REVIEW

File: ios/RNCallKeep/RNCallKeep.m [hunk 1]
```diff
@@ -391,7 +391,7 @@ + (CXProviderConfiguration *)getProviderConfiguration:(NSDictionary*)settings
     providerConfiguration.maximumCallsPerCallGroup = 1;
     providerConfiguration.supportedHandleTypes = [NSSet setWithObjects:[NSNumber numberWithInteger:CXHandleTypePhoneNumber], nil];
     if (settings[@"supportsVideo"]) {
-        providerConfiguration.supportsVideo = settings[@"supportsVideo"];
+        providerConfiguration.supportsVideo = [settings[@"supportsVideo"] boolValue];
```
Comment (kylekurz) [inline ios/RNCallKeep/RNCallKeep.m:394]: Probably want to add a comment here about what this was doing, since it seems unrelated to this change. It's ok to do that on the actual PR we send to Wazo, but don't just ignore it.

**Review by kylekurz — APPROVED** [review]

File: ios/RNCallKeep/RNCallKeep.m (hunk 1 above)
Comment (bhuangy) [reply ios/RNCallKeep/RNCallKeep.m:394]: Ok I'll add some bullet points when we make the PR? Or do you mean comment in code.

Comment (kylekurz) [reply ios/RNCallKeep/RNCallKeep.m:394]: Yeah, commenting on the PR summary about what you're addressing is fine.
